src/component/HenvendelseInput.tsx

```diff
@@ -40,7 +41,12 @@ function HenvendelseInput(props: Props) {
                 maxLength={5000}
                 tellerTekst={defaultTellerTekst}
             />
-            <Hovedknapp htmlType={'submit'} title="Send">
+            <Hovedknapp
+                htmlType={'submit'}
+                title="Send"
+                autoDisableVedSpinner
+                spinner={props.submitting ? true : false}
```

Comment:
props.submitting er en boolean, så trenger ikke sjekke? bare bruke direkte

Reply:
Hvis jeg tar vekk `? true : false` da protesterer IntelliJ typescript med  "*TS2322: Type 'Boolean' is not assignable to type 'boolean|undefined'.*" `False` er verdien også når `submitting` er `undefined` tenkes det tror jeg.

Reply:
Se kommentar om `boolean` i props. Vil tro det kan ha noe med saken å gjøre

Reply:
ja det var det